### PR TITLE
Creating a function to format Command help template

### PR DESCRIPTION
**Pull request by edentech88**
## Purpose:
Created functions to set cli.CustomCommandHelp and CustomSubCommandHelp

### CODE REVIEW

File: common/cli/app.go [hunk 1]
```diff
@@ -71,6 +71,16 @@ func (app *App) SetCustomAppHelpTemplate(appHelperTemplate string) {
...
+// SetSubcommandHelpTemplate sets custom SubcommandHelpTemplate to cli
+func SetSubcommandHelpTemplate(subcommandHelpTemplate string) {
+	cli.SubcommandHelpTemplate = subcommandHelpTemplate
+}
+
+// SetCommandHelpTemplate sets custom CommandHelpTemplate to cli
+func SetCommandHelpTemplate(commandHelpTemplate string) {
+	cli.CommandHelpTemplate = commandHelpTemplate
+}
+
```
Comment (levkoburburas) [inline common/cli/app.go:83]:
```suggestion
// SetCommandHelpTemplate sets custom HelpTemplate to command
func (cmd *Command) SetCustomHelpTemplate(commandHelpTemplate string) {
	cmd.CustomHelpTemplate = commandHelpTemplate
}
```

**Review by levkoburburas — CHANGES_REQUESTED** [review]

File: common/cli/app.go (hunk 1 above)
Comment (edentech88) [reply common/cli/app.go:83]: @levkoburburas I don't think it works. 
Looking at urfave, it should work, but the custom help template is not being displayed when I tried it.
FYI, here you go: https://github.com/pastelnetwork/pastel-utility/blob/misc/setting-up-gonod-cli-tool/cmd/init.go#L19-L21
```
initCommand := cli.NewCommand("init")

initCommand.CustomHelpTemplate = CommandHelpTemplate
Am I missing something?
```

Comment (edentech88) [reply common/cli/app.go:83]: @levkoburburas am I missing something?

Comment (levkoburburas) [reply common/cli/app.go:83]: I don't know what you are doing wrong (don't have time to figure out), but from [their code](https://github.com/urfave/cli/blob/45952a7d1b0e956db2099493fe1a42837ff5bd0d/help.go#L198-L201). It must work.

Comment (edentech88) [reply common/cli/app.go:83]: @levkoburburas let me debug then, and see where it goes wrong.

Comment (edentech88) [reply common/cli/app.go:83]: @levkoburburas did you get a chance to try it [here](https://github.com/pastelnetwork/pastel-utility/blob/misc/setting-up-gonod-cli-tool/cmd/init.go#L19-L21)?

Comment (levkoburburas) [reply common/cli/app.go:83]: @edentech88 I don't have enough time to fix code every PR I review. if something doesn't work, you have to find on your own. But if you are sure that this is the `urfave` issue, not in your code. Let me know I will check everything

Comment (edentech88) [reply common/cli/app.go:83]: @levkoburburas it's not `urfave` issue, but rather gonode/common/cli

Comment (levkoburburas) [reply common/cli/app.go:83]: It doesn't matter where `urfave` or `gonode/common/cli`, the main thing is not in your code. 
But if it still turns out that issue is in your code after spending time. I will have to leave negative feedback. I would advise you to double check your code.

**Pull request closed without merging**